user-journey-tests/helpers/testHelpers/destination.js

```diff
@@ -58,8 +59,9 @@ export const completeDestinationTaskOnFarm = async () => {
       townOrCity: 'The street',
       postcode: 'N11AA'
     },
-    reasonForMovementPage
+    maximumAnimalsPage
   )
+  await maximumAnimalsPage.inputTextAndContinue('550', reasonForMovementPage)
```

Comment:
This part of the journey is only valid under certain pre-conditions now - i.e. if the Origin is Market or Unrestricted farm.

Is it worth renaming the function to reflect?